src/pages/AnimeKO/style.less

```diff
@@ -0,0 +1,25 @@
+@import './../pages';
```

Comment:
The UI styling is broken. Black text on black background
![image](https://github.com/MALSync/MALSync/assets/12599334/5909847f-359e-4582-8e20-4c4202607b64)